qa-physics/notes/rgb_fa19.md

```diff
@@ -0,0 +1,44 @@
+# Run Group B, Fall 2019, Pass 2
+
+## Run monitoring
+
+> [!IMPORTANT]
+> Check any run-dependent settings in `qa-physics/monitorRead.groovy`, such as beam energy.
+
+We will use the `sidisdvcs` train. There are inbending and outbending data, which we'll
+combine to one "dataset" in `run-monitoring.sh`.
+
+First make sure all skim files are cached:
+```bash
+bin/run-monitoring.sh -d rgb_fa19_sidisdvcs --check-cache --flatdir --focus-physics \
+  /cache/clas12/rg-b/production/recon/fall2019/torus+1/pass2/v1/dst/train/sidisdvcs/ \
+  /cache/clas12/rg-b/production/recon/fall2019/torus-1/pass2/v1/dst/train/sidisdvcs/
```

Comment:
these directories both have overlapping sets of run numbers, but the files themselves are slightly different; we need to choose one file for each run.